### PR TITLE
8.1 netmgr

### PR DESCRIPTION
**Pull request by alviteri**

### CODE REVIEW

**Comment by jerpelea** [comment]
@oshmoun @luk1337

File: netmgrd.te [hunk 1]
```diff
@@ -26,33 +26,29 @@ unix_socket_connect(netmgrd, qmuxd, qmuxd)
...
-#Ignore if device loading for private IOCTL failed
-dontaudit netmgrd kernel:system { module_request };
+# Allow netutils usage
+domain_auto_trans(netmgrd, netutils_wrapper_exec, netutils_wrapper)
 
 allow netmgrd proc_net:file rw_file_perms;
 allow netmgrd netmgr_data_file:dir rw_dir_perms;
 allow netmgrd netmgr_data_file:file create_file_perms;
-allow netmgrd system_file:file execute_no_trans;
+allow netmgrd system_file:file { execute_no_trans lock };
```
Comment (luk1337) [inline netmgrd.te]: Do that to match CAF:
```
allow netmgrd system_file:file x_file_perms;
not_full_treble(`allow netmgrd system_file:file lock;')
```

Comment (alviteri) [reply netmgrd.te]: Ok thanks I will update soon .... Without this PR RIL is unknown for Loire on Enforcing ....

Comment (luk1337) [reply netmgrd.te]: How is this possible? netmgrd is only used for mobile data.

Comment (alviteri) [reply netmgrd.te]: Is possible ... If you want I can post screens xD before - after

**Comment by alviteri** [comment]
updated from caf tag

File: netmgrd.te (hunk 1 above)
Comment (oshmoun) [reply netmgrd.te]: I got ril on 8.1 and enforcing here on kugo,without this pr

Comment (alviteri) [reply netmgrd.te]: 6.4 kernel @oshmoun ?

**Comment by luk1337** [comment]
Are these netutils_wrapper rules necessary? CAF doesn't seem to have them.

**Comment by alviteri** [comment]
come on guys I posted the denials .... only on permissive ril worked here ... now works with enforcing

**Comment by alviteri** [comment]
I just picked the .te file from wahoo to fix them

**Comment by oshmoun** [comment]
Could it be that these denies only appear on first boot after a factory reset?
@davidteri91 did you ever boot the device with permissive selinux after the last time it was wiped?

**Comment by alviteri** [comment]
Factory reset not needed to see them ... But as I said these denials appears only with 6.4 stack

**Comment by alviteri** [comment]
screenshot_20180124-184655

**Comment by oshmoun** [comment]
I did test with 6.4,and i have ril there

**Comment by jerpelea** [comment]
please wait for v9 then we can decide

**Comment by jerpelea** [comment]
patch is needed by v9 release

**Pull request merged**